extensions/graphviz_support/graphviz_support/method/method.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from typing import List
+
+
+class Method:
```

Comment:
Shall we rename it as GraphVizNode instead of Method? Method keyword is also used in LabGraph core.

Reply:
Done